Components/typography.html

```diff
@@ -21,124 +21,98 @@
+
+            <div class="example-div">
+                <h1>This is a h1 heading.</h1>
+                <div class="horizontal-line-code"></div>
+                <h2>This is a h2 heading.</h2>
+                <div class="horizontal-line-code"></div>
+                <h3>This is a h3 heading.</h3>
+                <div class="horizontal-line-code"></div>
+                <h4>This is a h4 heading.</h4>
+                <div class="horizontal-line-code"></div>
+                <h5>This is a h5 heading.</h5>
+                <div class="horizontal-line-code"></div>
+                <h6>This is a h6 heading.</h6>
+                <div class="horizontal-line-code"></div>
+                <p>This is a paragraph.</p>
+            </div>
+
+
+            <div class="horizontal-line"></div>
+
+            <div class="content-heading">Font family</div>
+
+            <div class="content-subtitle">
+                Nuclei uses <span class="highlight-text">'Trebuchet MS'</span> which is inbuilt in Windows.
+            </div>
+
+            <iframe class="code-snippet"
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=css&width=737&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=font-family%253A%2520%27Trebuchet%2520MS%27%252C%2520%27Lucida%2520Sans%2520Unicode%27%252C%2520%27Lucida%2520Grande%27%252C%2520%27Lucida%2520Sans%27%252C%2520Arial%252C%2520sans-serif%252C%2520sans-serif%253B"
+                style="width: 900px; height: 225px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+
+            <div class="content-heading">Preview</div>
+
+            <div class="example-div">
+                <h1>This is an example of font-family <span class="highlight-text">Trebuchet MS</span></h1>
+                <div class="horizontal-line-code"></div>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Est nisi, ut ipsa ab vel consequuntur sequi
+                    nam nulla animi, deserunt amet repudiandae deleniti quod quos! Ratione, nobis ducimus. Totam
+                    laboriosam, odit nihil magni debitis nostrum ducimus voluptas velit! Necessitatibus officia
+                    quibusdam non odit est quam voluptatem, deserunt dolorum earum expedita autem ipsa nobis, sit quidem
+                    praesentium voluptas, tempora id veniam deleniti ipsam in quis reiciendis. Omnis, in ea ipsum velit
+                    necessitatibus quisquam a illum aut perferendis nulla fugit consequuntur incidunt, eius assumenda
+                    expedita cumque! Animi quis nostrum temporibus veniam tenetur alias asperiores sint. Aut laboriosam,
+                    iusto consectetur eos officiis et.</p>
+            </div>
+
+            <div class="horizontal-line"></div>
```

Comment:
for the horizontal line we have an hr tag also

Reply:
yes but I have used styling for it